te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
+In the screenshot above you can see the different areas of the Test-Editor-Web which are divided
+through the different yellow areas. The description of the specific area can be found under the number of this area underneath.   
+
+#### 1. The Test-Navigator area
+In this area the user can specify the structure of the different test artifacts.
```

Comment:
That seems a bit abstract, and its also not really the individual test artifacts that are structured, but the workspace as a whole. Maybe "workspace" needs to be introduced here as a concept, and related to something the user will likely know (i.e. the place he can store his files, and organize them in folders).